2024-05-25 13:36:43.403208Z jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "MIGRATION_MODEL" not found (this database is empty); SQL statement:
SELECT ID, VERSION FROM MIGRATION_MODEL ORDER BY UPDATE_TIME DESC [42104-224]
2024-05-25 13:36:46.002960Z jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "DATABASECHANGELOG" not found (this database is empty); SQL statement:
SELECT COUNT(*) FROM PUBLIC.DATABASECHANGELOG [42104-224]
2024-05-25 13:36:46.432323Z jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "DATABASECHANGELOGLOCK" not found (this database is empty); SQL statement:
SELECT COUNT(*) FROM PUBLIC.DATABASECHANGELOGLOCK [42104-224]
2024-05-25 13:36:46.467025Z jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "DATABASECHANGELOG" not found; SQL statement:
SELECT COUNT(*) FROM PUBLIC.DATABASECHANGELOG [42102-224]
2024-06-01 14:40:43.919787Z database: opening /opt/keycloak/data/h2/keycloakdb
org.h2.message.DbException: Error opening database: "Sleep interrupted" [8000-224]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.store.FileLock.getExceptionFatal(FileLock.java:429)
	at org.h2.store.FileLock.sleep(FileLock.java:424)
	at org.h2.store.FileLock.lockFile(FileLock.java:324)
	at org.h2.store.FileLock.lock(FileLock.java:110)
	at org.h2.engine.Database.<init>(Database.java:317)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:343)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:125)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:322)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:231)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:545)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:526)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:840)
Caused by: org.h2.jdbc.JdbcSQLNonTransientConnectionException: Error opening database: "Sleep interrupted" [8000-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:516)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 20 more
Caused by: java.lang.InterruptedException: sleep interrupted
	at java.base/java.lang.Thread.sleep(Native Method)
	at org.h2.store.FileLock.sleep(FileLock.java:422)
	... 17 more
2024-06-01 15:49:40.643779Z database: opening /opt/keycloak/data/h2/keycloakdb
org.h2.message.DbException: Error opening database: "Sleep interrupted" [8000-224]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.store.FileLock.getExceptionFatal(FileLock.java:429)
	at org.h2.store.FileLock.sleep(FileLock.java:424)
	at org.h2.store.FileLock.lockFile(FileLock.java:324)
	at org.h2.store.FileLock.lock(FileLock.java:110)
	at org.h2.engine.Database.<init>(Database.java:317)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:343)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:125)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:322)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:231)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:545)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:526)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:840)
Caused by: org.h2.jdbc.JdbcSQLNonTransientConnectionException: Error opening database: "Sleep interrupted" [8000-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:516)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 20 more
Caused by: java.lang.InterruptedException: sleep interrupted
	at java.base/java.lang.Thread.sleep(Native Method)
	at org.h2.store.FileLock.sleep(FileLock.java:422)
	... 17 more
2024-06-01 17:48:26.978169Z database: opening /opt/keycloak/data/h2/keycloakdb
org.h2.message.DbException: Error opening database: "Sleep interrupted" [8000-224]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.store.FileLock.getExceptionFatal(FileLock.java:429)
	at org.h2.store.FileLock.sleep(FileLock.java:424)
	at org.h2.store.FileLock.lockFile(FileLock.java:324)
	at org.h2.store.FileLock.lock(FileLock.java:110)
	at org.h2.engine.Database.<init>(Database.java:317)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:343)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:125)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:322)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:231)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:545)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:526)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:840)
Caused by: org.h2.jdbc.JdbcSQLNonTransientConnectionException: Error opening database: "Sleep interrupted" [8000-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:516)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 20 more
Caused by: java.lang.InterruptedException: sleep interrupted
	at java.base/java.lang.Thread.sleep(Native Method)
	at org.h2.store.FileLock.sleep(FileLock.java:422)
	... 17 more
2024-06-01 17:50:35.377263Z database: opening /opt/keycloak/data/h2/keycloakdb
org.h2.message.DbException: Error opening database: "Sleep interrupted" [8000-224]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.store.FileLock.getExceptionFatal(FileLock.java:429)
	at org.h2.store.FileLock.sleep(FileLock.java:424)
	at org.h2.store.FileLock.lockFile(FileLock.java:334)
	at org.h2.store.FileLock.lock(FileLock.java:110)
	at org.h2.engine.Database.<init>(Database.java:317)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:343)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:125)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:322)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:231)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:545)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:526)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:840)
Caused by: org.h2.jdbc.JdbcSQLNonTransientConnectionException: Error opening database: "Sleep interrupted" [8000-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:516)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 20 more
Caused by: java.lang.InterruptedException: sleep interrupted
	at java.base/java.lang.Thread.sleep(Native Method)
	at org.h2.store.FileLock.sleep(FileLock.java:422)
	... 17 more
2024-06-01 17:50:54.379394Z database: flush
org.h2.message.DbException: General error: "org.h2.mvstore.MVStoreException: Writing to sun.nio.ch.FileChannelImpl@6ecc1660 failed; length 12288 at 45056 [2.2.224/2]" [50000-224]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:407)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:122)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:1546)
	at org.h2.mvstore.MVStore.panic(MVStore.java:371)
	at org.h2.mvstore.FileStore.storeBuffer(FileStore.java:1527)
	at org.h2.mvstore.FileStore.lambda$serializeAndStore$3(FileStore.java:1432)
	at org.h2.mvstore.FileStore.submitOrRun(FileStore.java:1401)
	at org.h2.mvstore.FileStore.serializeAndStore(FileStore.java:1432)
	at org.h2.mvstore.FileStore.lambda$storeIt$2(FileStore.java:1376)
	at org.h2.mvstore.FileStore.submitOrRun(FileStore.java:1401)
	at org.h2.mvstore.FileStore.storeIt(FileStore.java:1375)
	at org.h2.mvstore.MVStore.storeNow(MVStore.java:843)
	at org.h2.mvstore.MVStore.store(MVStore.java:808)
	at org.h2.mvstore.MVStore.commit(MVStore.java:778)
	at org.h2.mvstore.MVStore.commit(MVStore.java:770)
	at org.h2.mvstore.tx.TransactionStore.close(TransactionStore.java:365)
	at org.h2.mvstore.db.Store.close(Store.java:349)
	at org.h2.engine.Database.closeOpenFilesAndUnlock(Database.java:1293)
	at org.h2.engine.Database.<init>(Database.java:401)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:343)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:125)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:322)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:231)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:545)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:526)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:840)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "org.h2.mvstore.MVStoreException: Writing to sun.nio.ch.FileChannelImpl@6ecc1660 failed; length 12288 at 45056 [2.2.224/2]" [50000-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 34 more
Caused by: org.h2.mvstore.MVStoreException: Writing to sun.nio.ch.FileChannelImpl@6ecc1660 failed; length 12288 at 45056 [2.2.224/2]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.DataUtils.writeFully(DataUtils.java:478)
	at org.h2.mvstore.SingleFileStore.writeFully(SingleFileStore.java:67)
	at org.h2.mvstore.SingleFileStore.writeFully(SingleFileStore.java:28)
	at org.h2.mvstore.RandomAccessStore.writeChunk(RandomAccessStore.java:366)
	at org.h2.mvstore.RandomAccessStore.writeChunk(RandomAccessStore.java:28)
	at org.h2.mvstore.FileStore.storeBuffer(FileStore.java:1524)
	... 28 more
Caused by: java.nio.channels.ClosedChannelException
	at java.base/sun.nio.ch.FileChannelImpl.ensureOpen(FileChannelImpl.java:159)
	at java.base/sun.nio.ch.FileChannelImpl.write(FileChannelImpl.java:854)
	at org.h2.mvstore.DataUtils.writeFully(DataUtils.java:474)
	... 33 more
2024-06-02 08:14:05.753330Z jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientConnectionException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:690)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1425)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.checkClosed(JdbcXAConnection.java:485)
	at org.h2.jdbc.JdbcConnection.rollback(JdbcConnection.java:511)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.close(JdbcXAConnection.java:455)
	at org.h2.jdbcx.JdbcXAConnection.close(JdbcXAConnection.java:76)
	at io.agroal.pool.ConnectionHandler.closeConnection(ConnectionHandler.java:178)
	at io.agroal.pool.ConnectionPool$DestroyConnectionTask.run(ConnectionPool.java:799)
	at io.agroal.pool.ConnectionPool.close(ConnectionPool.java:200)
	at io.agroal.pool.DataSource.close(DataSource.java:79)
	at io.quarkus.agroal.runtime.DataSources.stop(DataSources.java:449)
	at io.quarkus.agroal.runtime.DataSources_Bean.doDestroy(Unknown Source)
	at io.quarkus.agroal.runtime.DataSources_Bean.destroy(Unknown Source)
	at io.quarkus.agroal.runtime.DataSources_Bean.destroy(Unknown Source)
	at io.quarkus.arc.impl.AbstractInstanceHandle.destroyInternal(AbstractInstanceHandle.java:78)
	at io.quarkus.arc.impl.ContextInstanceHandleImpl.destroy(ContextInstanceHandleImpl.java:26)
	at io.quarkus.arc.impl.AbstractSharedContext.destroy(AbstractSharedContext.java:102)
	at io.quarkus.arc.impl.ArcContainerImpl.shutdown(ArcContainerImpl.java:465)
	at io.quarkus.arc.Arc.shutdown(Arc.java:65)
	at io.quarkus.arc.runtime.ArcRecorder$1.run(ArcRecorder.java:53)
	at io.quarkus.runtime.StartupContext.runAllInReverseOrder(StartupContext.java:84)
	at io.quarkus.runtime.StartupContext.close(StartupContext.java:73)
	at io.quarkus.runner.ApplicationImpl.doStop(Unknown Source)
	at io.quarkus.runtime.Application.stop(Application.java:208)
	at io.quarkus.runtime.Application.stop(Application.java:155)
	at io.quarkus.runtime.ApplicationLifecycleManager$ShutdownHookThread.run(ApplicationLifecycleManager.java:437)
2024-06-02 08:14:05.923905Z jdbc[4]: exception
org.h2.jdbc.JdbcSQLNonTransientConnectionException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:690)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1425)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.checkClosed(JdbcXAConnection.java:485)
	at org.h2.jdbc.JdbcConnection.rollback(JdbcConnection.java:511)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.close(JdbcXAConnection.java:455)
	at org.h2.jdbcx.JdbcXAConnection.close(JdbcXAConnection.java:76)
	at io.agroal.pool.ConnectionHandler.closeConnection(ConnectionHandler.java:178)
	at io.agroal.pool.ConnectionPool$DestroyConnectionTask.run(ConnectionPool.java:799)
	at io.agroal.pool.ConnectionPool.close(ConnectionPool.java:200)
	at io.agroal.pool.DataSource.close(DataSource.java:79)
	at io.quarkus.agroal.runtime.DataSources.stop(DataSources.java:449)
	at io.quarkus.agroal.runtime.DataSources_Bean.doDestroy(Unknown Source)
	at io.quarkus.agroal.runtime.DataSources_Bean.destroy(Unknown Source)
	at io.quarkus.agroal.runtime.DataSources_Bean.destroy(Unknown Source)
	at io.quarkus.arc.impl.AbstractInstanceHandle.destroyInternal(AbstractInstanceHandle.java:78)
	at io.quarkus.arc.impl.ContextInstanceHandleImpl.destroy(ContextInstanceHandleImpl.java:26)
	at io.quarkus.arc.impl.AbstractSharedContext.destroy(AbstractSharedContext.java:102)
	at io.quarkus.arc.impl.ArcContainerImpl.shutdown(ArcContainerImpl.java:465)
	at io.quarkus.arc.Arc.shutdown(Arc.java:65)
	at io.quarkus.arc.runtime.ArcRecorder$1.run(ArcRecorder.java:53)
	at io.quarkus.runtime.StartupContext.runAllInReverseOrder(StartupContext.java:84)
	at io.quarkus.runtime.StartupContext.close(StartupContext.java:73)
	at io.quarkus.runner.ApplicationImpl.doStop(Unknown Source)
	at io.quarkus.runtime.Application.stop(Application.java:208)
	at io.quarkus.runtime.Application.stop(Application.java:155)
	at io.quarkus.runtime.ApplicationLifecycleManager$ShutdownHookThread.run(ApplicationLifecycleManager.java:437)
